project-metrics/metrics_service/metrics/release_granularity.py

```diff
@@ -0,0 +1,68 @@
+"""Release Granularity metric."""
+
+import logging
+import sqlalchemy
+from typing import Text, Dict
+
+from database import db
+from database import models
+from metrics import base
+
+
+class ReleaseGranularityMetric(base.Metric):
+  """A metric tracking the average number of commits between releases."""
+
+  def _format_value(self, avg_commits: float) -> Text:
+    return '%d c/r' % round(avg_commits)
+
+  def _score_value(self, avg_commits: float) -> models.MetricScore:
+    if avg_commits > 1000:
+      return models.MetricScore.CRITICAL
+    elif avg_commits > 500:
+      return models.MetricScore.POOR
+    elif avg_commits > 100:
+      return models.MetricScore.MODERATE
+    elif avg_commits > 10:
+      return models.MetricScore.GOOD
+    else:
+      return models.MetricScore.EXCELLENT
+
+  def _compute_value(self) -> float:
+    """Computes the average number of commits between releases over the last 90
+
+    days.
+
+    Considering only production releases, we can just count all commits
+    committed between the first and last release of the 90-day window, and
+    divide be the number of releases (excluding the last one).
+
+    Raises:
+      ValueError: if less than two releases exist.
+
+    Returns:
+      The percentage of passing builds.
+    """
+    logging.info('Counting commits per release')
+    session = db.Session()
+    releases = session.query(models.Release).filter(
+        models.Release.is_last_90_days()).all()
+    release_count = len(releases)
+
+    if release_count == 0:
```

Comment:
You can just keep the `< 2` case, the error message is correct for both ;)

Reply:
Done.